nci60/nbconverted/0.download_nci60.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Download NCI-60 Growth Inhibition Data
+# 
+# https://wiki.nci.nih.gov/display/NCIDTPdata/NCI-60+Growth+Inhibition+Data
+# 
+# 71 Cell Lines treated with 57,000 compounds
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+
+import sys
+
+sys.path.append("../")
+from utils import download_utils
+
+
+# In[2]:
+
+
+# Construct output directory
+output_dir = pathlib.Path("data")
+output_dir.mkdir(exist_ok=True)
+
+
+# In[3]:
+
+
+# Base url for downloading NCI-60 data
+nci_url = "https://wiki.nci.nih.gov/download/attachments/"
+
+
+# In[4]:
+
+
+# Download the dose response data
+attachment_id = "147193864"
+attachment_name = "DOSERESP.zip"
+version = "6"
+modification_date = "1672801037000"
+api = "v2"
```

Comment:
Consider using a more specific name here, for ex. `api_version` or similar. It also may be worth adding greater specificity to the `version` above, for ex. `data_version`.

Reply:
yes, this is a good idea. Will also update in the function. Thanks!